### ISSUE TITLE
wrong results

### ISSUE THREAD

**Issue by yug125lk** [JuliaWolleb/Diffusion-based-Segmentation#57]
Hi, thank you again for sharing this great work. I used the code for image-to-image translation. When I tested the model, it gave me a different output (completely different from the input). I thought the model was overfitted. I have 11870 folders.  I set up the flags as follows:
MODEL_FLAGS="--image_size 128 --num_channels 128 --class_cond False --num_res_blocks 2 --num_heads 1 --learn_sigma True --use_scale_shift_norm False --attention_resolutions 16"
DIFFUSION_FLAGS="--diffusion_steps 1000 --noise_schedule linear --rescale_learned_sigmas False --rescale_timesteps False"
TRAIN_FLAGS="--lr 1e-4 --batch_size 4"

I changed the dropout, learning rate, reduced diffusion steps, and reduced num_channels. I also tried with other datasets. but got the same wrong results. Could you suggest how to solve this problem? and thank you.
